EIPS/eip-3074.md

```diff
@@ -0,0 +1,179 @@
+---
+eip: 3074
+title: Sponsored Transaction Precompile
+author: Sam Wilson (@SamWilsn)
+discussions-to: <URL>
+status: Draft
+type: Standard
+category: Core
+created: 2020-10-15
+---
+
+## Simple Summary
+
+Creates a new precompile, analogous to `CALL`, that sets `msg.sender` based on an ECDSA signature.
+
+## Abstract
+
+We propose adding two precompiles:
+ - `CALL_PRECOMPILE` - forwards a `CALL`, setting `msg.sender` according to an ECDSA signature, using a relayer-sponsee nonce for replay protection.
+ - `NONCE_PRECOMPILE` - provides access to relayer-sponsee nonces expected by `CALL_PRECOMPILE`.
+
+## Motivation
+
+Sponsored transactions—the separation of fee payment from transaction content—have been a long standing feature request. Unlike similar proposals, this EIP specifies a method of implementing sponsored transactions that allows both externally owned accounts (EOAs) and [EIP-2938] contracts to act as sponsors.
+
+With the explosion of tokens built on Ethereum, especially stable coins, it has become common for EOAs to hold valuable assets without holding any Ether at all. These assets must be converted to Ether before they can be used to pay gas fees, but without Ether to pay for the conversion, it's impossible to convert them. Sponsored transactions break the circular dependency.
+
+While it is possible to emulate sponsored transactions (ex. [Gas Station Network][gsn]), these solutions require specific support in callee contracts.
+
+## Specification
+
+We propose adding two precompiles to implement sponsored transactions:
```

Comment:
EIPs don't propose, they assert.  The target audience for an EIP is people 5 to 10 years in the future, after this EIP has been final for years.